src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CaseActionsForCaseWorkerControllerTest.java

```diff
@@ -1069,6 +1073,15 @@ public void judgmentSubmittedError400() throws Exception {
                 .andExpect(status().isBadRequest());
     }
 
+    @Test
+    public void dynamicFixCaseAPIError400() throws Exception {
+        mvc.perform(post(DYNAMIC_FIX_CASE_API_URL)
+                .content("error")
+                .header("Authorization", AUTH_TOKEN)
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isBadRequest());
```

Comment:
You can improve the 400/500 error unit tests by also checking a service wasn't called. See mockito's verify/never
e.g. something like this
`verify(fixCaseApiService, never()).checkUpdateMultipleReference(any(CaseDetails.class), anyString());`

Reply:
Updated